src/common.jl

```diff
@@ -10,17 +12,22 @@ export  A_mul_B
 #### Type-handling
 """
     Adivtype(A, b)
+If A is a function map then:
+`typeof(one(eltype(b)))`
 Determine type of the division of an element of `b` against an element of `A`:
 `typeof(one(eltype(b))/one(eltype(A)))`
 """
-Adivtype(A, b) = typeof(one(eltype(b))/one(eltype(A)))
+Adivtype(A, b) = isa(A, FunctionMap) ? typeof(one(eltype(b))) : typeof(one(eltype(b))/one(eltype(A)))
```

Comment:
Actually, isn't it here? 

https://github.com/Jutho/LinearMaps.jl/blob/master/src/LinearMaps.jl#L8

eltype should work then since `FunctionMap{T,F1,F2}<:AbstractLinearMap{T}`